server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -220,6 +245,7 @@ export class ScribeLambda implements IPartitionLambda {
                                 } else {
                                     const nackMessage = summaryResponse.message as ISummaryNack;
                                     await this.sendSummaryNack(nackMessage);
+                                    lumberJackMetric?.setProperties({[CommonProperties.clientSummarySuccess]: false});
```

Comment:
Do we also need to do `lumberJackMetric?.error(errorMsg);` here?

Reply:
Had a discussion offline with Henrique. Since, this metrics covers only the success or failure of the lamdba handler, the summary failure is not considered a failure of this metric. This will get reflected in the summary metric.